### PR TITLE
Notify when create subject

### PR DESCRIPTION
**Pull request by tuanna2704**

### CODE REVIEW

File: app/views/users/shared/_links.html.haml [hunk 1]
```diff
@@ -6,8 +6,7 @@
   = link_to "Sign up", new_registration_path(resource_name)
   %br
 
-- if devise_mapping.recoverable? && controller_name != 'passwords' &&
-  controller_name != 'registrations'
+- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations'
```
Comment (taddd1993) [inline app/views/users/shared/_links.html.haml]: ' '-> " "

**Comment by taddd1993** [comment]
OK

File: app/mailers/subject_notifier.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class SubjectNotifier < ApplicationMailer
+  default from: "any_from_address@example.com"
+
+  def send_create_subject_email user, subject
+    @user = user
+    @subject = subject
+    mail( to: @user.email,
```
Comment (dieunb) [inline app/mailers/subject_notifier.rb]: Why not one of line?

File: config/application.rb [hunk 3]
```diff
@@ -23,5 +23,6 @@ class Application < Rails::Application
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
     config.assets.paths << Rails.root.join("app", "assets", "fonts")
+    config.autoload_paths << Rails.root.join('services')
```
Comment (dieunb) [inline config/application.rb]: double quote

**Comment by taddd1993** [comment]
OK!

**Pull request merged**